src/components/vm/disks/vmDisksCard.tsx

```diff
@@ -171,45 +169,25 @@ export const VmDisksCard = ({
     vm,
     vms,
     disks,
-    renderCapacity,
     supportedDiskBusTypes,
     storagePools,
     onAddErrorNotification
 } : {
     vm: VM,
     vms: VM[],
     disks: VMDiskWithData[],
-    renderCapacity: boolean,
     supportedDiskBusTypes: string[],
     storagePools: StoragePool[],
     onAddErrorNotification: (notification: Notification) => void,
 }) => {
     const [openActions, setOpenActions] = useState(new Set());
-    let renderCapacityUsed = false;
-    let renderAccess = false;
-    let renderAdditional = false;
     const columnTitles: (string | ListingTableColumnProps)[] = [_("Device")];
```

Comment:
Similar to networking, repeating "Type: disk" in the "Disks" table feels boring and wasteful. This should be something identifying, like the source -- a qcow file, a pool name, iscsi name, etc.

Reply:
Yes, we have the Source. The type is mostly interesting to distinguish cd-roms from regular disks, I think. If we have space, I'd leave it, just to call out cd-roms loudly.

Reply:
I changed my mind: the "Eject" and "Insert" buttons are enough to tell the user what's going on. So let's remove the "Device" column.

Reply:
Nice!